### PR TITLE
#65

### PR DESCRIPTION
**Pull request by smiska**

### CODE REVIEW

File: examples/knob.html [hunk 1]
```diff
@@ -398,7 +421,7 @@
 
 	<h2>knob-paged-list</h2>
 
-	<knob-paged-list params="store: store, fields: ['id', 'email', 'name', 'title'], search: 'title', orderBy: {name: -1}, sort: [{label: 'By id', value:'id'}, {label: 'By name', value:'name'}]">
+	<knob-paged-list params="store: store, fields: ['id', 'email', 'name', 'title'], search: 'title', orderBy: {name: -1}, sort: [{label: 'By id', value:'id'}, {label: 'By name', value:'name'}], labels: {noResults: 'No results found'}">
```
Comment (gyulanemeth) [inline examples/knob.html]: Why did you add this to the example?

File: examples/knob.js [hunk 2]
```diff
@@ -56,6 +56,9 @@ initKnob({
 	},
 	icons: {
 		search: "#icon-search"
+	},
```
Comment (gyulanemeth) [inline examples/knob.js]: Why did you add this?

File: examples/knob.min.css [hunk 3]
```diff
@@ -212,6 +212,30 @@
   position: relative;
   text-align: left; }
 
+/* $width: 60px;
```
Comment (gyulanemeth) [inline examples/knob.min.css]: please remove the commented code

File: README.md [hunk 4]
```diff
@@ -42,6 +42,32 @@ click     | function | No           |               | This is the callback which
...
+## knob-toggleSwitch
+
+This switch component is used for enabling or disabling features. If you click on it, you toggle the value property associated with the component.
+
+### Params
+
+Param     | Type                   | Required     | Default value         | Description
+---       |---                     |---           |---                    |---
+value     | ko.observable(boolean) | Yes          | ko.observalble(false) | With this observable, you can set the default state of the switch button.
+variation | string                 | No           | "square"              | By giving it to the component you can set the switch button rectangular.
```
Comment (gyulanemeth) [inline README.md]: it seems from the example that "square" is not the default value.

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "knob-js",
-  "version": "0.1.30",
+  "version": "0.1.32",
```
Comment (gyulanemeth) [inline package.json]: .31? did you not merge the master branch into it? if not, then do so.

File: spec/toggleSwitch/toggleSwitchSpec.js [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+describe("==== ToggleSwitch ====", function(){
+
+
+	describe(" - with invalid config", function(){
+
+		var mockBase = {};
+
+		var createToggleSwitch = toggleSwitchCore({
+				ko: ko,
```
Comment (gyulanemeth) [inline spec/toggleSwitch/toggleSwitchSpec.js]: tabs!

File: src/toggleSwitch/style.js [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+	var colors = theme;
+
+	var factor2 = 5;
+	var factor3 = 10;
+	var factor4 = 15;
+	var factor5 = 20;
+	var factor6 = 25;
+
+	function createColorShades(color) {
+		var color1 = tinycolor(color);
```
Comment (gyulanemeth) [inline src/toggleSwitch/style.js]: please remove all of the unused color variables. after that remove all of the unused factor variables, etc.

File: examples/knob.html (hunk 1 above)
Comment (smiska) [inline examples/knob.html]: it should have been an 'accidental remove, then re-add' type of modification

File: examples/knob.js (hunk 2 above)
Comment (smiska) [inline examples/knob.js]: There was a comma missing, which I added.
